src/layouts/Header/DesktopMenu.tsx

```diff
@@ -12,34 +9,8 @@ import AdminLink from './AdminLink'
 import Dropdown from './Dropdown'
 import SeatDetails from './SeatDetails'
 
-interface URLParams {
-  provider: string
-}
-
 export function LoginPrompt() {
-  const { provider } = useParams<URLParams>()
-
   const to = window.location.href
-  const { pathname } = useLocation()
-
-  if (!provider) return null
```

Comment:
These two conditionals are no longer relevant since moving over to the login specific layout here: https://github.com/codecov/gazebo/blob/782e9b4909d395c9ed3fbbf1831a3ca3595bf18b/src/App.tsx#L65-L74